Makefile

```diff
@@ -653,10 +654,6 @@ ${PROOFTHEORIES}: %_proofs.thy: %.pbf ${BF_GEN_PATH} ${STATICSOURCES} \
 # Preprocessed source files
 ###########################
 
-%.s: %.S ${GENHEADERS} ${STATICHEADERS} | ${DIRECTORIES}
-	@echo " [CPP] $@"
-	$(Q)${CPP} ${CPPFLAGS} -CC -E -o $@ $<
-
```

Comment:
The lowercase .s was referenced in a previous rule (where you changed the .s to .S). The idea is that this build step runs the .S files through the C pre-processor, producing .s files that are passed to the assembler.